optionals/compat_ace/CfgVehicles.hpp

```diff
@@ -0,0 +1,53 @@
+class CfgVehicles {
+    class ModuleEmpty_F;
+    class ModuleMine_F: ModuleEmpty_F {
+        function = QEFUNC(modules,bi_moduleMine);
+    };
+
+    // Hide ACE Zeus modules for which replacements exist
+    class ACEGVAR(zeus,moduleBase);
+
+    class ACEGVAR(zeus,moduleEditableObjects): ACEGVAR(zeus,moduleBase) {
```

Comment:
The spacing can be removed, afaik the project avoids this spacing in config files.